### PR TITLE
Repo clean-up and Rubocop action

### PR DESCRIPTION
**Pull request by mikker**
It looks like we miss Rubocop's warnings if we don't have them here. This action annotates directly on Github which is a big help.

I also disabled a bunch of the checks in the `Metrics/*Length` category as we've discussed earlier.

### CODE REVIEW

File: bin/dev [hunk 1]
```diff
@@ -28,7 +28,7 @@ USER_ID_GROUP = %w[u g].map { |f| `id -#{f}`.chomp }.join(':')
 RUBY_IMAGE = options.fetch(:image, 'ruby:latest')
 FRAMEWORKS = options.fetch(:frameworks, 'rails,sinatra,grape')
 
-IMAGE_PATH_SAFE = RUBY_IMAGE.gsub(':', '_')
+IMAGE_PATH_SAFE = RUBY_IMAGE.tr(':', '_')
```
Comment (estolfo) [inline bin/dev]: Why did you change this?

File: lib/elastic_apm/central_config/cache_control.rb [hunk 2]
```diff
@@ -26,7 +26,7 @@ def initialize(value)
       def parse!(value)
         value.split(',').each do |token|
           k, v = token.split('=').map(&:strip)
-          instance_variable_set(:"@#{k.gsub('-', '_')}", v ? v.to_i : true)
+          instance_variable_set(:"@#{k.tr('-', '_')}", v ? v.to_i : true)
```
Comment (estolfo) [inline lib/elastic_apm/central_config/cache_control.rb]: and this?

File: spec/integration/rails_grape_spec.rb [hunk 3]
```diff
@@ -11,7 +11,8 @@
 if enabled
   require 'action_controller/railtie'
 
-  RSpec.describe 'Rails and Grape integration', :mock_intake, :allow_running_agent do
+  RSpec.describe 'Rails and Grape integration', :mock_intake,
+    :allow_running_agent do
```
Comment (estolfo) [inline spec/integration/rails_grape_spec.rb]: can we line this up under `:mock_intake`?

File: bin/dev (hunk 1 above)
Comment (mikker) [reply bin/dev]: `rubocop-performance` suggests that it's faster than `gsub` for regular string (non-regex)

File: lib/elastic_apm/central_config/cache_control.rb (hunk 2 above)
Comment (mikker) [reply lib/elastic_apm/central_config/cache_control.rb]: same 😊

File: spec/integration/rails_grape_spec.rb (hunk 3 above)
Comment (mikker) [reply spec/integration/rails_grape_spec.rb]: We can break both? I don't like special situation indenting levels, really.

File: spec/integration/rails_grape_spec.rb [hunk 4]
```diff
@@ -11,7 +11,8 @@
 if enabled
   require 'action_controller/railtie'
 
-  RSpec.describe 'Rails and Grape integration', :mock_intake, :allow_running_agent do
+  RSpec.describe 'Rails and Grape integration', :mock_intake,
```
Comment (mikker) [inline spec/integration/rails_grape_spec.rb]:
```suggestion
  RSpec.describe 'Rails and Grape integration',
    :mock_intake, :allow_running_agent do
```

File: spec/integration/rails_grape_spec.rb (hunk 3 above)
Comment (mikker) [inline spec/integration/rails_grape_spec.rb]:
```suggestion
```

File: bin/dev (hunk 1 above)
Comment (estolfo) [reply bin/dev]: oh nice, I didn't know that it was faster.
